docs/data-sources/ip_sla_track_list.md

```diff
@@ -0,0 +1,76 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_ip_sla_track_list"
+sidebar_current: "docs-aci-data-source-aci_ip_sla_track_list"
+description: |-
+  Data source for ACI IP SLA Track List
+---
+
+# aci_ip_sla_track_list #
+
+Data source for ACI IP SLA Track List
+
+## API Information ##
+
+* Class: [fvTrackList](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTrackList/overview)
+
+* Supported in ACI versions: 4.1(1i) and later.
+
+* Distinguished Name Format: `uni/tn-{name}/tracklist-{name}`
+
+## GUI Information ##
+
+* Location: `Tenants -> Policies -> Protocol -> IP SLA -> Track Lists`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_ip_sla_track_list" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "test_name"
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+* `name` (name) - (string) The name of the IP SLA Track List object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the IP SLA Track List object.
+* `annotation` (annotation) - (string) The annotation of the IP SLA Track List object.
+* `description` (descr) - (string) The description of the IP SLA Track List object.
+* `name_alias` (nameAlias) - (string) The name alias of the IP SLA Track List object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `percentage_down` (percentageDown) - (string) The down percentage of the IP SLA Track List to determine the route to be considered available or unavailable. This attribute is only applicable when the track type is 'percentage'.
+* `percentage_up` (percentageUp) - (string) The up percentage of the IP SLA Track List to determine the route to be considered available or unavailable. The percentage up must be greater than percentage down. This attribute is only applicable when the track type is 'percentage'.
+* `type` (type) - (string) The type of the IP SLA Track List.
+* `weight_down` (weightDown) - (string) The down weight assigned to the IP SLA Track List to determine the route to be considered available or unavailable. This attribute is only applicable when the track type is 'weight'.
```

Comment:
Shall we change this to?
->
The down weight assigned to the IP SLA track list determines whether a route is considered available or unavailable. This attribute is applicable only when the track type is 'weight'.